### DISCUSSION TITLE
open currently opened file on server startup

### DISCUSSION THREAD

**Discussion by soredake** [yandeu/five-server-vscode#29] [Q&A]
It will be nice if there is option to do this.

**Comment by yandeu** [comment]
In VSCode, you can simply right-click on the file and "open with five-server".

**Reply by soredake** [reply]
This option is fine, but requires me doing two clicks instead of one (if this option will be implemented).

**Reply by yandeu** [reply]
So basically you want the current file to open once you click the bottom right button?

**Reply by soredake** [reply]
Yes, that's how another [live server addon](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) works.

**Reply by yandeu** [reply]
Oh, I didn't know. I guess I can add it as well :)
